mem0/memory/main.py

```diff
@@ -266,6 +272,23 @@ def get_all(self, user_id=None, agent_id=None, run_id=None, limit=100):
             }
             for mem in memories[0]
         ]
+        
+        if self.version == "v1.1":
```

Comment:
Same here as well if user doesn't pass version in config then it will always be `v1` and not `v1.1`

Reply:
the default behavior is without-graphRAG